ctapipe/tools/apply_models.py

```diff
@@ -176,14 +202,30 @@ def _apply(self, reconstructor):
             for tel_id, table in chunk.items():
                 tel = self.loader.subarray.tel[tel_id]
 
-                if tel not in reconstructor._models:
-                    self.log.warning(
-                        "No model in %s for telescope type %s, skipping tel %d",
-                        reconstructor,
-                        tel,
-                        tel_id,
-                    )
-                    continue
+                if isinstance(reconstructor, DispReconstructor):
+                    if tel not in reconstructor._norm_models:
+                        self.log.warning(
```

Comment:
It cannot really be the case that only one is missing, right?

Reply:
Yes. I changed it to only check for a regressor.